src/common/components/cards/CardTable.js

```diff
@@ -62,7 +62,7 @@ export default class CardTable extends React.Component {
       selectable={!this.props.selectable || card.source !== 'builtin'}
     >
       <TableRowColumn width={130}>{card.name}</TableRowColumn>
-      <TableRowColumn width={70}>{typeToString(card.type)}</TableRowColumn>
```

Comment:
Hmm one of the bigger improvements in MUI v1 is the new Table API. Might be worth switching